pkg/arvo/sys/vane/gall.hoon

```diff
@@ -2,1457 +2,2882 @@
-      pyl/(map bone mark)                               ::  req'd translations
-      zam/scar                                          ::  opaque ducts
-  ==                                                    ::
-++  sofa                                                ::  queue for blocked
-  $:  kys/(qeu (trel duct prey club))                   ::  queued kisses
-  ==                                                    ::
-++  stic                                                ::  statistics
-  $:  act/@ud                                           ::  change number
-      eny/@uvJ                                          ::  entropy
-      lat/@da                                           ::  time
-  ==                                                    ::
---                                                      ::
-    ::::::::::::::::::::::::::::::::::::::::::::::::::::::  vane header
-    ::::::::::::::::::::::::::::::::::::::::::::::::::::::
+::
+::  (rest of arvo)
+::
+|%
+::
+::  +internal-gift: synonym for +cuft.
+::
+++  internal-gift  cuft
+::
+::  +internal-task: synonym for +cush.
+::
+++  internal-task  cush
+::
+::  +agent-action: synonym for +club.
+::
+++  agent-action  club
+::
+::  +coke: cook.
+::
+++  coke
+  $?  %inn
+      %out
+      %cay
+  ==
+::
+::  +volt: voltage.
+::
+++  volt  ?(%low %high)
+::
+::  +security-control: security control.
+::
+++  security-control  $@(?(%iron %gold) [%lead p=ship])
+::
+::  +reverse-ames: reverse ames message.
+::
+++  reverse-ames
+  $%
+      :: diff
+      ::
+      [action=%d p=mark q=*]
+      ::  etc.
+      ::
+      [action=%x ~]
+  ==
+::
+::  +forward-ames: forward ames message.
+::
+++  forward-ames
+  $%
+      :: message
+      ::
+      [action=%m mark=mark noun=*]
+      :: "peel" subscribe
+      ::
+      [action=%l mark=mark path=path]
+      :: subscribe
+      ::
+      [action=%s path=path]
+      :: cancel+unsubscribe
+      ::
+      [action=%u ~]
+  ==
+::
+::  +foreign-response: foreign response.
+::
+++  foreign-response
+  $?  %peer
+      %peel
+      %poke
+      %pull
+  ==
+--
+::
+::  (local arvo)
+::
+|%
+::
+::  +internal-note: +ap note.
+::
+++  internal-note
+  $%  [task=%meta =term =vase]
+      [task=%send =ship =internal-task]
+  ==
+::
+::  +internal-move: agent-level move.
+::
+::  Analogous to an Arvo move, except these are routed by bone, instead of
+::  duct.
+::
+++  internal-move
+  $:
+    =bone
+    move=(wind internal-note internal-gift)
+  ==
+::
+::  +move: Arvo-level move.
+::
+++  move  (pair duct (wind note-arvo gift-arvo))
+--
+::
+::  (%gall state)
+::
+|%
+::
+::  +gall-old: upgrade path.
+::
+++  gall-old  ?(gall)
+::
+::  +gall: all state.
+::
+++  gall
+  $:
+      :: state version
+      ::
+      %0
+      :: agents by ship
+      ::
+      =agents
+  ==
+::
+::  +subscribers: subscriber data.
+::
+++  subscribers
+  $:
+      :: incoming subscribers
+      ::
+      incoming=bitt
```

Comment:
it would be nice to have an alias for `bitt` and `boat` to maintain consistency of the naming conventions